db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_02_08_195724) do
+ActiveRecord::Schema.define(version: 2019_02_15_183324) do
```

Comment:
@cdoyle-temple I don't think any of these schema chnages need to be part of this PR, do they?

Reply:
no.

Reply:
I think they are there because I built on previous work thinking it would be merged first. They don't need to be here.